plugins/Morpheus/stylesheets/general/_default.less

```diff
@@ -119,17 +119,18 @@ body {
   font-size: 1em;
 }
 
-a {
-  color: #255792;
-  text-decoration: underline;
-  cursor: pointer;
-}
-
 #root {
   margin: 0 10px;
   position: relative;
 }
 
+a {
+ color: @theme-color-link;
+ text-decoration: none;
```

Comment:
I'm torn here whether we should have kept it as underlined as it was, does it help anything removing that now and re-add it later?

Reply:
There had been one global rule defining it as underlined. And another rule in a css file that isn't included in the installation css, defining it as not underlined. This PR unifies both to one global rule. Reducing that now will make it easier to change the link styles later globally.